### PR TITLE
[ONNX CI] ONNX CI fixes

### PR DESCRIPTION
**Pull request by mchrusci**
* Fix build status "Aborted" on tox tests fail
* Prepare for cleanup

### CODE REVIEW

File: .ci/onnx/jenkins/Jenkinsfile.groovy [hunk 1]
```diff
@@ -174,8 +174,12 @@ def main(String label, String projectName, String projectRoot, String dockerCont
                     RunToxTests(configurationMaps)
                 }
             }
-            catch(hudson.AbortException e) {
-                currentBuild.result = 'ABORTED'
+            catch(e) {
```
Comment (postrational) [inline .ci/onnx/jenkins/Jenkinsfile.groovy]: Let's delete `Jenkinsfile.groovy` altogether, if it's no longer needed.

File: .ci/onnx/jenkins/Jenkinsfile [hunk 2]
```diff
@@ -166,6 +174,13 @@ def main(String label, String projectName, String projectRoot, String dockerCont
                     RunToxTests(configurationMaps)
                 }
             }
+            catch(e) {
+                if ("$e".contains("143")) {
```
Comment (postrational) [inline .ci/onnx/jenkins/Jenkinsfile]: Please add a comment explaning the reson for using "143" here.

File: .ci/onnx/jenkins/utils/utils.groovy [hunk 3]
```diff
@@ -86,10 +86,12 @@ def CreateStage(String stageName, Closure method, configurationMaps) {
         try {
             def prepareEnvMap = GenerateMap(genericBodyMethod, configurationMaps)
             parallel prepareEnvMap
-        } catch(hudson.AbortException e) {
-            AbortedException(e)
         } catch(e) {
-            Exception(e)
+            if ("$e".contains("143")) {
```
Comment (postrational) [inline .ci/onnx/jenkins/utils/utils.groovy]: Please add a comment explaning the reson for using "143" here.

File: .ci/onnx/jenkins/Jenkinsfile.groovy (hunk 1 above)
Comment (mchrusci) [reply .ci/onnx/jenkins/Jenkinsfile.groovy]: O want to delete it in next PR, to make sure the switch is smooth.

Comment (postrational) [reply .ci/onnx/jenkins/Jenkinsfile.groovy]: OK

File: .ci/onnx/jenkins/utils/utils.groovy (hunk 3 above)
Comment (mchrusci) [reply .ci/onnx/jenkins/utils/utils.groovy]: Done

File: .ci/onnx/jenkins/Jenkinsfile (hunk 2 above)
Comment (mchrusci) [reply .ci/onnx/jenkins/Jenkinsfile]: Done

**Pull request merged**